### PR TITLE
As 345 sources directory

### PR DESCRIPTION
**Pull request by nensidosari**
### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://as-345-sources-directory.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx [hunk 1]
```diff
@@ -88,7 +103,7 @@ const Image = (props: SourceAuthorProps) => {
   );
 };
 
-const UserHighlight = (props: SourceAuthorProps) => {
+export const UserHighlight = (props: UserHighlightProps): ReactElement => {
```
Comment (rebelchris) [inline packages/shared/src/components/widgets/PostUsersHighlights.tsx:106]: I was confused by this file changes (based on name)
Maybe we should abstract some of these to different components?

(not blocking ofcourse)

File: packages/shared/src/graphql/sources.ts [hunk 2]
```diff
@@ -82,6 +82,7 @@ export interface Source {
   headerImage?: string;
   color?: string;
   description?: string;
+  createdAt?: Date;
```
Comment (rebelchris) [inline packages/shared/src/graphql/sources.ts:85]: Probably not exposed for this one right?
We can just query based on it and not return it.

**Review by rebelchris — APPROVED** [review]
Awesome, think we can about merge it and solve from there?

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:106]: I thought of it as well as this component is too big and does to many stuff.

File: packages/shared/src/graphql/sources.ts (hunk 2 above)
Comment (nensidosari) [reply packages/shared/src/graphql/sources.ts:85]: yeah true we won't need to expose it, just query by it would be enough

**Pull request merged**